### PR TITLE
Drop python2 support

### PR DESCRIPTION
**Pull request by aronsky**
As per #206, remove Python 2 support.

Note: I haven't completely tested the win32 compilation on Wine. I updated the download to use the latest Python 3 available as an MSI package (since later Python 3 versions fail to install successfully under Wine), but `make` fails due to other errors that I didn't feel like diving into. If anyone knows how the build of the w32 Python bindings works and can test its functionality, that'll be great.

### CODE REVIEW

**Comment by XVilka** [comment]
I also think we should drop everything that older 3.6: https://repology.org/project/python3/versions

File: python/test-r_bp.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python2
+#!/usr/bin/env python
```
Comment (XVilka) [inline python/test-r_bp.py]: Shouldn't we use `/usr/bin/env python3` for now? Just in case.

File: python-wrapper [hunk 2]
```diff
@@ -3,24 +3,31 @@
...
+	python3
+	python38
+	python3.8
+	python37
+	python3.7
+	python36
+	python3.6
+	python35
+	python3.5
+	python34
```
Comment (XVilka) [inline python-wrapper]: drop old 3.x versions maybe?

File: mp.py [hunk 3]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python2
+#!/usr/bin/env python
```
Comment (XVilka) [inline mp.py]: python3 here too

File: python/test-r_bp.py (hunk 1 above)
Comment (aronsky) [reply python/test-r_bp.py]: Sure, will fix.

**Comment by aronsky** [comment]
> I also think we should drop everything that older 3.6: https://repology.org/project/python3/versions

I'm all for newer versions, but dropping older versions of Python 3 seems unnecessary - the only difference is the call to `Py_Finalize` (which is `void` before 3.6), that's easy to maintain... And if the wine build works, it's a lot easier to build with Python 3.4 (which can be installed under wine without problems), than newer versions (which aren't distributed in MSI packaging, and their installation fails).

**Comment by aronsky** [comment]
The latest commit now builds `lang_python.{so|dylib}`, instead of `lang_python3.{so|dylib}`.

I guess that once this gets merged, we should also update the r2pm repo to include just lang_python, instead of lang_python2 and lang_python3.

However, this switch will require a manual update from an older version of lang_python3 to the new lang_python. Is that OK?

**Comment by XVilka** [comment]
Yes, I think that's OK. Thank you!

**Comment by XVilka** [comment]
Merging this to let people test and catch bug early.

**Pull request merged**